packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -183,6 +185,33 @@ contract TaikoL1 is EssentialContract, IHeaderSync, TaikoEvents {
         });
     }
 
+    /**
+     * Revert the chain to a previously verified block and also update
+     * certain internal state variables.
+     */
+    function revertToBlock(
+        uint64 latestVerifiedHeight,
+        uint64 latestVerifiedId,
+        bytes[] calldata inputs
+    ) external {
+        // Only owner or a emergency-handling multisig admin can
+        // authorize this transaction.
+        require(
+            msg.sender == owner() ||
+                msg.sender == AddressResolver(this).resolve("admin"),
```

Comment:
I think this is first concept of "admin". Do we want to allow other functions that are typically "onlyOwner" to be called by admin as well, so we can keep owner account safer? If so can we put this in a modifier "onlyOwnerOrAdmin" ?